client/components/thank-you/index.tsx

```diff
@@ -84,10 +99,12 @@ const ThankYouNextStep = ( props: ThankYouNextStepProps ) => {
 
 	return (
 		<React.Fragment key={ stepKey }>
-			<h3>{ stepTitle }</h3>
-			<div>
-				<p>{ stepDescription }</p>
-				<div>{ stepCta }</div>
+			<div className="thank-you__steps">
+				<div>
+					<h3>{ stepTitle }</h3>
+					<p>{ stepDescription }</p>
+				</div>
+				<div className="thank-you__steps-buttons">{ stepCta }</div>
```

Comment:
As above, I don't think `steps` is correct.

Also, it may be easier to call this `steps-cta` to match the incoming variable/prop name.
```suggestion
				<div className="thank-you__step-cta">{ stepCta }</div>
```